src/reducers/followerReducer.js

```diff
@@ -0,0 +1,62 @@
+import {
+  followType,
+  unfollowType,
+  getFollowersType,
+  getFollowingType,
+} from '../actions/followAction';
+import initialState from '../store/initialState';
+
+const { follow } = initialState;
+
+/**
+ * User followers reducer function
+ *
+ * @param {object} [state=follow] - The initial state to load
+ * @param {object} action - The action to dispatch
+ * @param {string} action.type - The action type
+ * @param {object|boolean} action.payload - The action payload
+ * @returns {object} - Returns the mutated state object
+ */
+export default (state = follow, { type, payload }) => {
+  switch (type) {
+    case followType.loading:
+      return {
+        ...state,
+        isLoading: payload
+      };
+    case followType.success:
+      return {
+        ...state,
+        following: [
+          ...state.following,
+          payload
+        ],
+        isLoading: false,
+      };
+    case followType.failure:
+      return {
+        ...state,
+        isLoading: false
+      };
+    case unfollowType.success:
+      return {
+        ...state,
+        following: state.following.filter(user => user.id !== payload.id),
+        isLoading: false,
+      };
+    case getFollowersType.success:
```

Comment:
@devdbrandy I can see you have getFollowersType.success and getFollowingType.success. They are both returning the same thing. Can we do away with one of them?

Reply:
Thanks for pointing that @fejiroofficial, getFollowersType.success is mutating the `followers` key while getFollowingType.success is `following`.

Reply:
okay, thanks for the clarity @devdbrandy